auth-web/src/views/auth/home/HomeView.vue

```diff
@@ -565,8 +565,4 @@ export default class HomeView extends Vue {
     background-color: transparent;
   }
 
-  .hero-banner-login-btn {
```

Comment:
This css is no longer needed it was only changing the one buttons text to the wrong color.